README.md

```diff
@@ -1,153 +1,232 @@
 <img src="data/logo/toolbox-logo-landscape.svg" alt="Toolbox logo landscape" width="800"/>
 
-[Toolbox](https://github.com/containers/toolbox) is a tool that offers a
-familiar package based environment for developing and debugging software that
-runs fully unprivileged using [Podman](https://podman.io/).
-
-The toolbox container is a fully *mutable* container; when you see
-`yum install ansible` for example, that's something you can do inside your
-toolbox container, without affecting the base operating system.
-
-This is particularly useful on
-[OSTree](https://ostree.readthedocs.io/en/latest/) based operating systems like
-[CoreOS](https://coreos.fedoraproject.org/) and
-[Silverblue](https://silverblue.fedoraproject.org/).  The intention of these
-systems is to discourage installation of software on the host, and instead
-install software as (or in) containers.
+[![Zuul](https://zuul-ci.org/gated.svg)](https://softwarefactory-project.io/zuul/t/local/builds?project=containers/toolbox)
+[![Daily Pipeline](https://img.shields.io/badge/zuul-periodic-informational)](https://softwarefactory-project.io/zuul/t/local/builds?project=containers%2Ftoolbox&pipeline=periodic)
 
-However, this tool doesn't *require* using an OSTree based system — it
-works equally well if you're running e.g. existing Fedora Workstation or
-Server, and that's a useful way to incrementally adopt containerization.
+[![Fedora package](https://img.shields.io/fedora/v/toolbox/rawhide)](https://src.fedoraproject.org/rpms/toolbox/)
+[![Arch Linux package](https://img.shields.io/archlinux/v/community/x86_64/toolbox)](https://www.archlinux.org/packages/community/x86_64/toolbox/)
```

Comment:
I took the liberty to split out the badges into a [separate pull request](https://github.com/containers/toolbox/pull/589), while I try to rebase this PR on top of `master`.